components/canvas/Card/Card.css

```diff
@@ -16,6 +16,7 @@ diamond-card {
 
   &[radius] {
     border-radius: var(--diamond-radius);
+    overflow: hidden;
```

Comment:
Why?

Reply:
Otherwise the content doesn't get clipped